cmd/openshift-install/create.go

```diff
@@ -198,6 +198,12 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 
 		err := runner(rootOpts.dir)
 		if err != nil {
+			// Attempt to gather logs if cluster asset creation failed
+			if cmd.Name() == "cluster" {
+				if err2 := runGatherBootstrapCmd(rootOpts.dir); err2 != nil {
```

Comment:
So if the terraform fails we will see the tf failure then trying to gather bootstrap and then failure of failing to gather due to reasons like we never reaches the cluster creation in the first place.

So i think we need to be more selective of when we execute this,
- maybe based on a specific error type?
- or only when the tf state was present 
or something like that.